utils/io/fileutils/files_test.go

```diff
@@ -50,10 +50,14 @@ func TestGetFileOrDirPathFile(t *testing.T) {
 	}
 
```

Comment:
I suggets we add one more test to this function, in which the file is expected not exist:
```
	if exists {
		t.Error("File go.mod is supposed to be missing, but it exists.")
	}
```

Reply:
Done I've added a test also for a file that shouldn't be found.